static/scss/dashboard.scss

```diff
@@ -184,14 +188,15 @@ c.validation-wrapper {
           }
 
           .dashboard-button {
-            background-color: $mm-brand-bg;
+            background-color: $mm-brand-bg !important;
```

Comment:
@gsidebo here we are using this css on a  button which was imported from ``react-mdl/lib/Button``. This is to override basic style of mentioned lib.